src/assets/images/EmptyBoxPlainImage.tsx

```diff
@@ -0,0 +1,43 @@
+export const EmptyBoxPlainImage = (
```

Comment:
nit: can we rename this to `EmptyWhiteBoxImage` and rename the original `EmptyBoxImage` to `EmptyBlueBoxImage` for clarity?

Reply:
Good point! Adjusted in de025d5.